src/pages/settings/settings.html

```diff
@@ -73,7 +73,7 @@ <h4>Learn where you can spend your crypto today.</h4>
         </ion-note>
       </button>
       <button ion-item class="create-button" *ngIf="walletsGroups && walletsGroups[0]" (click)="goToAddView()" translate>
-        Create or Import a Key
```

Comment:
I'm not sure on removing "Import"

Reply:
in the ticket was specified to delete the import option of `configuration -> create or import a key`... if it  won't possible to import, so the label must change

Reply:
Fixed! Thanks for reviewing it